packages/dds/map/src/index.ts

```diff
@@ -98,3 +98,46 @@ export const SharedMap = {
  * This alias is for legacy compat from when the SharedMap class was exported as public.
  */
 export type SharedMap = ISharedMap;
+
+/**
+ * {@inheritDoc ISharedDirectory}
```

Comment:
Inheriting doc seems inappropriate here. Something like this seems better:

`Entrypoint for {@link ISharedDirectory} creation`.

However, I think once Craig's PR here merges: https://github.com/microsoft/FluidFramework/pull/20028, you'll have a clearer way to document using `ISharedObjectKind` (and can just inherit docs from that)

Reply:
Yeah, just referencing `ISharedDirectory` is fine. inheritdoc has a lot of weird quirks in our toolchain IIRC.